common/src/main/java/org/jboss/da/common/util/FileUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.common.util;
```

Comment:
We are already using appache commons and I don't think replicating code from library is great idea.